xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/BitfinexExchange.java

```diff
@@ -58,8 +63,15 @@ public void remoteInit() throws IOException, ExchangeException {
     List<CurrencyPair> currencyPairs = dataService.getExchangeSymbols();
     exchangeMetaData = BitfinexAdapters.adaptMetaData(currencyPairs, exchangeMetaData);
 
+    // Get the last-price of each pair. It is needed to infer XChange's priceScale out of Bitfinex's pricePercision
+    org.knowm.xchange.bitfinex.v2.service.BitfinexMarketDataServiceRaw dataServiceV2 =
+            new org.knowm.xchange.bitfinex.v2.service.BitfinexMarketDataServiceRaw(this);
+    Map<CurrencyPair, BigDecimal> lastPrices = Arrays.stream(dataServiceV2.getBitfinexTickers(currencyPairs))
+    	.map(org.knowm.xchange.bitfinex.v2.BitfinexAdapters::adaptTicker)
+    	.collect(Collectors.toMap(t -> t.getCurrencyPair() , t -> t.getLast()));
```

Comment:
we are using v2 API in the integration with the v1 API ?

Reply:
v2